pkg/installer/const.go

```diff
@@ -70,12 +70,12 @@ const (
 	nodeDriverRegistrarContainerName = "node-driver-registrar"
 	csiProvisionerContainerName      = "csi-provisioner"
 
-	// "csi-provisioner:v2.1.0"
-	csiProvisionerContainerImage = "csi-provisioner@sha256:4ca2ce98430ca0b87d5bc1a6d116ecdf1619cfe6db693d8d5aa438f6821e0e80"
-	// "livenessprobe:v2.1.0"
-	livenessProbeContainerImage = "livenessprobe@sha256:6f056a175ff4ead772edc9bf99aef74c275a83c51868dd26090dcb623425a742"
-	// "csi-node-driver-registrar:v2.1.0"
-	nodeDriverRegistrarContainerImage = "csi-node-driver-registrar@sha256:9f9ce5c98e44d66b8ad34351616fdf78765b9f24c3c3b496cee784dadf63f528"
+	// "csi-provisioner:v2.2.0"
+	csiProvisionerContainerImage = "csi-provisioner@sha256:3b465cbcadf7d437fc70c3b6aa2c93603a7eef0a3f5f1e861d91f303e4aabdee"
```

Comment:
we have two references to the same constant. Shall we remove the ones in this grouping here, and rely on only the ones at the top?

In the interest of time, i'll merge this and follow up with another PR for the fix